src/config/AuthorizationManager.h

```diff
@@ -6,6 +6,7 @@
 #include <juce_data_structures/juce_data_structures.h>
 #include "Resources.h"
 
+#ifndef IMAGIRO_SKIP_AUTH
```

Comment:
Why gate the class definition behind this flag and not remove the #include of this header file entirely? Actually that's what I initially tried, but there seems to be a transitive dependency chain between this file being included somehow, with `Resources.h` successfully resolving the `FileBackedPreset` class.

Cursor suggested I split the `Resources.h` into a header file and `Resources.cpp` implementation file, but that seemed a little out of my comfort zone to make that call, so I opted for the minimally intrusive change here which is to leave the imports as they are and only gate the class definition itself.

Works well enough.